moto/mediapackage/exceptions.py

```diff
@@ -1 +1,13 @@
 from __future__ import unicode_literals
+
+from moto.core.exceptions import JsonRESTError
+
+
+class MediaPackageClientError(JsonRESTError):
+    code = 400
+
+
+# AWS service exceptions are caught with the underlying botocore exception, ClientError
+class ClientError(MediaPackageClientError):
```

Comment:
Moto intercepts any HTTP requests that are made. Whatever error is thrown by us, will be converted into a HTTP 400 response. Boto3 will in turn throw a ClientError for any 400-error that is returned.

TLDR: The name doesn't matter here. Considering it's purpose, it's probably cleaner to make it a ChannelNotFoundError (with a hardcoded message)

Reply:
My production code has fragments like this:
```
 try:
            return self.aws_client.update_origin_endpoint(**params)
        except ClientError as error:
            if 'NotFoundException' == error.response['Error']['Code']:
                Logger.info(f'OriginEndpoint {params} was not found')
            else:
                Logger.info(f'Errorr updating media package with params {params} :  {error}')

            raise error
```

And it works nicely catching Boto errors. So my purpose is to make Moto throw the exact error as Boto does. I am not sure if there's a way to make it work if we don't use the same error naming.